examples/stable_diffusion_xl/gm/modules/embedders/open_clip/model.py

```diff
@@ -133,7 +133,7 @@ def __init__(
         self.ln_final = text.ln_final
         self.text_projection = text.text_projection
 
-        self.attn_mask = Parameter(text.attn_mask, requires_grad=False)
```

Comment:
Is this modification verified on 910A/901B and in ms2.1 and ms2.2? I suppose the reason to initiate attn_mask as a Parameter instead of a Tensor is that 910B requires that.

Reply:
Tried on both 910A (MS2.1) and 910B (MS2.2) in both PyNative and Graph modes, and it worked without any problems. Here's the test script, let me know if it's invalid:
<details><summary>Test script</summary>

```python
import mindspore as ms

from gm.modules.embedders.modules import FrozenOpenCLIPEmbedder2

ms.set_context(mode=ms.GRAPH_MODE, device_target="Ascend")

model = FrozenOpenCLIPEmbedder2(
    "ViT-bigG-14-Text", layer="penultimate", always_return_pooled=True, legacy=False, require_pretrained=False
)

sample, _ = model.tokenize("This is a test sentence.")
sample = ms.Tensor(sample)

output = model(sample)
print(output[0].shape, output[1].shape)
```
</details>

Reply:
The test script looks fine to me.

I also made this modification and then trained sd-xl with text encoders on 910B. It has not raised any error. I think this modification is fine.